service/src/test/java/io/camunda/service/license/CamundaLicenseTest.java

```diff
@@ -114,7 +117,7 @@ public void shouldReturnProperValidValuesWhenLicenseIsSaas() throws InvalidLicen
     testLicense.initializeWithLicense(TEST_LICENSE);
 
     // then
-    assertEquals(SAAS_LICENSE_TYPE, testLicense.getLicenseType());
+    assertEquals(LicenseType.SAAS, testLicense.getLicenseType());
```

Comment:
Is this not a breaking API change? Previously, we would get a lowercased string, as per the API specification. Now with these enums, would it not be the upper-case? Please correct me if I am wrong here

Reply:
A lower case is still returned in the API. If you look at `LicenseController`, its setting the returned value with the `getName()` method which is returning the lower-case representation of the enum

`LicenseControllerTest` is also testing that a lower case is being returned

Reply:
Thanks for explaining! 👍